app/src/lib/components/Player/Fullscreen.svelte

```diff
@@ -681,6 +681,9 @@
 		@media screen and (max-width: 719px) {
 			max-height: 28vh;
 		}
+		@media screen and (min-width: 1800) {
```

Comment:
If you could kindly make this small change (add in px):
```css
@media screen and (min-width: 1800px) {
```

then i'll merge! :+1: